esp/src/src/nls/fr/hpcc.ts

```diff
@@ -658,6 +670,7 @@ export = {
     Results: "Résultat(s)",
     Resume: "reprendre",
     RetainSuperfileStructure: "Conserver la structure du Superfichier",
+    RetainSuperfileStructureReason: "Retenir le structure de superfile doit être permis quand plus qu'un sous-fichier existe",
```

Comment:
I meant to say superfichier not superfile

Reply:
Change made.